docs/samples/browser-plugin-meetings/app.js

```diff
@@ -380,7 +380,7 @@ meetingsListElm.onclick = (e) => {
   selectedMeetingId = e.target.value;
   const meeting = webex.meetings.getAllMeetings()[selectedMeetingId];
 
-  if(meeting && meeting.passwordStatus === 'REQUIRED'){
+  if(meeting && meeting.passwordStatus === 'REQUIRED' && meeting.passwordStatus !== 'UNKNOWN'){
```

Comment:
`&& meeting.passwordStatus !== 'UNKNOWN'` will always be true because we first already check that passwordStatus is 'REQUIRED'. `UNKNOWN` means that we haven't had a response from the "get meeting info" backend api yet, so I was thinking that we should disable the join button, not sure about the password related elements, maybe they should be disabled too...

Reply:
Thanks for pointing out understood, 
Then condition `else if(meeting.passwordStatus === 'REQUIRED')` will be good to have 
In this case, we will 
disable the join button as well as password elements.

Reply:
@marcin-bazyl, have incorporated the comments, Please check once. Thanks.